### PR TITLE
Refactor cron values in Tasks and fix the cron file deletion

### PR DESCRIPTION
**Pull request by stephdl**
This pull request introduces several enhancements and fixes related to cron task management:

- Bug Fix: Resolved an issue where cron files were not being removed when a task was deleted.
- Disabled State: Added a disabled state in the modal during the save operation to improve user feedback.
- Slider Feature: Implemented a slider in the cron task interface, allowing system administrators to easily select the desired time for task scheduling.
- UI Changes: All changes related to the cron task are confined to the user interface; the backend functionality remains unchanged.

https://github.com/user-attachments/assets/e8334f32-4cd0-4a73-b2ea-ec53e44fef54

https://github.com/NethServer/dev/issues/7036

### CODE REVIEW

File: ui/src/views/Tasks.vue [hunk 1]
```diff
@@ -389,8 +389,21 @@ export default {
...
+        let cronValue = task.cron;
+        let cron_enabled = false;
+        if (cronValue.endsWith("h")) {
+          cronValue = String(parseInt(cronValue) * 60); // Convert hours to minutes and ensure it's a string
+          cron_enabled = true;
+        } else if (cronValue.endsWith("m")) {
+          cronValue = String(parseInt(cronValue)); // Remove the 'm', keep the number, and ensure it's a string
+          cron_enabled = true;
+        } else {
+          cronValue = String(0); // Ensure it's a string if it's already in the desired format
```
Comment (andre8244) [inline ui/src/views/Tasks.vue:403]:
```suggestion
          cronValue = "0";
```

**Review by andre8244 — COMMENTED** [review]
In the video a glitch appears: the first time "Save" is clicked there is a validation error on "Local user" combobox :thinking:

**Review by andre8244 — APPROVED** [review]
LGTM, but the glitch in the video (see previous comment) still worries me :man_shrugging: But we can fix it in another PR

**Comment by stephdl** [comment]
the glitch should have been fixed

File: imageroot/actions/delete-task/20delete [hunk 2]
```diff
@@ -14,5 +14,6 @@ data = json.load(sys.stdin)
 task_id = data["task_id"]
 localuser = data["localuser"]
 
-for file in glob.iglob("imapsync/"+localuser+'_'+task_id+".*"):
+# remove the task cron/env/password/lock files
+for file in glob.iglob("*/"+localuser+'_'+task_id+".*"):
```
[the remark this answers is not in the document]
Comment (stephdl) [reply imageroot/actions/delete-task/20delete:18]: simply files in state/cron  are not removed

**Review by andre8244 — APPROVED** [review]

File: imageroot/actions/delete-task/20delete (hunk 2 above)
[the remark this answers is not in the document]
Comment (stephdl) [reply imageroot/actions/delete-task/20delete:18]: we have two mounts 

```
/state/imapsync -> /etc/imapsync
/state/cron -> /etc/cron.d
```

**Pull request merged**